tools/codegen/dest/register_dispatch_key.py

```diff
@@ -87,8 +87,18 @@ def gen_structured(self, g: NativeFunctionsGroup) -> List[str]:
 
     @method_with_native_function
     def gen_unstructured(self, f: NativeFunction) -> Optional[str]:
+        inplace_meta = False
         if self.dispatch_key not in f.dispatch:
-            return None
+            if (self.dispatch_key == DispatchKey.Meta and
+                    f.func.kind() is SchemaKind.inplace and
+                    # Defer to composites for meta implementation
+                    DispatchKey.CompositeImplicitAutograd not in f.dispatch and
+                    DispatchKey.CompositeExplicitAutograd not in f.dispatch and
```

Comment:
This seems like the morally correct behavior. But if your goal is to quickly get meta tensors working for all in-place ops, won't this probably cause in-place ops to fail in a couple of cases?

Random example: `atan` hasn't been ported to structured yet. `atan_` is currently a `CompositeExplicitAutograd` op, since it redispatches to `atan_out`. So the codegen won't generate a dummy meta kernel for `atan_`- it'll redispatch to `atan_out`, which will complain about not having a meta kernel.

Is it fair to say that **all** inplace meta kernels really just return `self` + do some error checking, regardless of whether or not they're composite?

Reply:
or if this is just meant to be a temporary crutch to get prove out the MetaTensor-module design and most of the in-place needed already work, then this seems fine

Reply:
I actually had not thought about this, but you're right, this will cause some ops that might have worked to not work. I suppose we can pray that these never get hit during module initializer (and if they do, port them to structured). But if this really seems like it's going to be a widespread problem I'm going to have to come up with some other strategy with dealing with these. Anyone fancy using dependency graphs to decide if a composite is usable or not for a backend? :P